spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -112,6 +112,7 @@ def argv(self):
         else:
             kernel_cmd = [
                 pyexec,
+                '-Xfrozen_modules=off',
```

Comment:
```suggestion
                # This is necessary to avoid a spurious message on Windows.
                # Fixes spyder-ide/spyder#20800.
                '-Xfrozen_modules=off',
```